### PR TITLE
User friendly search in permission page

### PR DESCRIPTION
**Pull request by sarathmohan4**

### CODE REVIEW

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/DirectoryService.java [hunk 1]
```diff
@@ -191,4 +192,49 @@ public DirectoryGroup mapFromAttributes(Attributes attr) throws NamingException
 		});
 	}
 
+	/**
+	 * Method to query ladp with displayName.
```
Comment (sivanag1974) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/DirectoryService.java:196]: Search the LDAP with displayName

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/DirectoryService.java [hunk 2]
```diff
@@ -191,4 +192,49 @@ public DirectoryGroup mapFromAttributes(Attributes attr) throws NamingException
...
+		users.setData(usersList);
+		return users;
+	}
+
+	/**
+	 * Method to query ladp with displayName.
+	 * @param ntId
+	 * @return
+	 */
+	public ResponseEntity<DirectoryObjects> searchUserInGSM(String ntId) {
```
Comment (sivanag1974) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/DirectoryService.java:231]: searchByDisplayNameAndId

**Review by sivanag1974 — APPROVED** [review]

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/DirectoryService.java (hunk 1 above)
Comment (sarathmohan4) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/DirectoryService.java:196]: Done

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/DirectoryService.java (hunk 2 above)
Comment (sarathmohan4) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/DirectoryService.java:231]: Done

**Pull request merged**